### ISSUE TITLE
Geogebra app draws focus when loaded

### ISSUE THREAD

**Issue by dbrianwalton** [PreTeXtBook/pretext#1086]
Current behavior can be seen in Sample Article HTML when viewing Section 14 (Interactive Elements, Authored). Once the page finishes loading, the view immediately jumps to Figure 14.23 (Geogebra: from scratch). I have confirmed this in both Chrome and Safari.

In a modified version of the section, if I rearrange the figures, the focus is always drawn to the same Figure. If the figure is removed from the source, there is no jumping.

Why it happens: The GGBApplet seems actively requests the page focus, which bubbles through the iFrame. It only appears to be happening when the GeoGebra object definitions are established through javascript commands (hence, the from scratch).

Solution: Add another default parameter for the GGBApplet to prevent focus.
(pull-request is being prepared)

**Comment by kcrisman** [comment]
> Current behavior can be seen in Sample Article HTML when viewing Section
> 14 (Interactive Elements, Authored). Once the page finishes loading, the
> view immediately jumps to Figure 14.23 (Geogebra: from scratch). I have
> confirmed this in both Chrome and Safari.
>

Thank you for discovering that; I had been noticing that all week, but
failed to see the significance - I thought I had been accidentally clicking
on a too-sensitive touchpad!

**Comment by dbrianwalton** [comment]
Unfortunately, the focus attribute may not be enough to fix this. (My page was just taking quite a bit longer for the focus to jump, so I thought I had fixed it.)

**Comment by dbrianwalton** [comment]
This question on the GeoGebra forums suggests that maybe the source is a zoomability on the interactive.

[https://help.geogebra.org/topic/scrolling-through-a-web-page-with-geogebra-applets-geogebra-captures-focus]

**Comment by dbrianwalton** [comment]
Okay, the offending command is the GeoGebra command `setPerspective`, which is the first command in the from-scratch javascript GGB interactive. The change of page focus appears to be an unintentional side effect by GeoGebra (it does not respect the focus flag) and seems to be out of our control.

On the other hand, there appears to be no loss of functionality by dropping that particular command from the script. The view for this interactive shows the graphical view by default.

**Comment by rbeezer** [comment]
Thanks, Brian!  Maybe @Alex-Jordan has more to add?

**Comment by Alex-Jordan** [comment]
Thanks Brian for figuring this out. In general, `setPerspective` will be an important thing to use when building GGB applets from scratch. The behavior and Brian's findings sound like it's a bug with GeoGebra, and so then I think we just report it to GeoGebra and let them look into it if they can.

With this applet, I would assume that not setting the perspective to `G`, you would still have the "algebra panel" showing (the panel that lists all the constructions). It wouldn't affect the usability of the applet, but it would be a distraction. Hiding that would have been my reason to use `setPerspective` here.

**Comment by rbeezer** [comment]
Resolved at #1905.
